hub/integrations/pytorch/dataset.py

```diff
@@ -50,7 +51,7 @@ def cast_type(tensor: np.ndarray):
     return tensor
 
 
-def _process(tensor, transform: Optional[Callable]):
+def _process(tensor, transform: Optional[Union[PytorchTransformFunction, Callable]]):
```

Comment:
Can we make it in a way, so there is no `Callable` passed ? Wrap it in `TransformFunction` ?

Reply:
done